lib/spack/spack/spec.py

```diff
@@ -1154,9 +1154,12 @@ def _libs_default_handler(spec: "Spec"):
     )
 
     for shared in search_shared:
-        # Since we are searching for link libraries, on Windows search only for
-        # ".Lib" extensions by default as those represent import libraries for implicit links.
-        libs = fs.find_libraries(name, home, shared=shared, recursive=True, runtime=False)
+        # Set runtime=False: we want find libraries used for linking by default
+        # Set max_depth=1: we default to searching the root directory and all the
+        #     directories underneath it, but we don't recurse beyond that by default
```

Comment:
`max_depth=1` is too low for multi-arch subdirs like on Debian: `prefix: /usr` -> libs are in `/usr/lib/x86_64-linux-gnu`.

I don't care so much about the literal `/` prefix (which would require depth 3 on Debian?) since that sounds like a configuration error.

Reply:
2 sounds good.